frontend/sumo-championship/src/components/molecules/CategoryForm/AgeCategoryFrom.tsx

```diff
@@ -0,0 +1,153 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import React, { useEffect, useState } from 'react';
+import TextField from '../TextField';
+import MinMaxField from '../MinMaxField';
+import Button from '../../Atoms/Button';
+import SubtitleWithIcon from '../../Atoms/SubtitleWithIcon';
+import { Category } from '../../../types/Seasons';
+
+const errorPointsValues = {
+  categoryName: 2,
+  gender: 3,
+  age: 5,
+};
+
+type props = {
+  editedCategoryNumber: number;
+  onSave: (categories: Category[]) => void;
+  categories: Category[];
+};
+
+const checkIfNameIsUnique = (array: Category[], name: string) => {
+  return !array.some((category) => category.ageName === name);
+};
+
+const AgeCategoryFrom: React.FC<props> = ({
+  onSave,
+  categories,
+  editedCategoryNumber,
+}) => {
+  const [categoryName, setCategoryName] = useState('');
+  const [errorPoints, setErrorPoints] = useState(1);
+  const [minAge, setMinAge] = useState(6);
```

Comment:
why hardcoded random default values?

Reply:
It was like it before, thanks to it user will know min and max values